header/p2p/options.go

```diff
@@ -92,6 +90,16 @@ type ClientParameters struct {
 	MaxRequestSize uint64
 	// MaxHeadersPerRequest defines the max amount of headers that can be requested per 1 request.
 	MaxHeadersPerRequest uint64
+	// GC defines the duration after which the peerTracker starts removing peers.
+	GC time.Duration
```

Comment:
As I mentioned some time ago, it would be good to extract gc and rely on it from peerTracker or @distractedm1nd PR or  backoff connector.  And in this case we will have to specify how much time do we need for our gc cycle.

Reply:
It's unrelated to GC times though.

Reply:
I mean, the abstraction is not related to the question: should we make GC time configurable or not? Is this something that brings value, or just bloats the config? 

I feel it doesn't bring anything and can be a constant.